Source/Core/Constants+Style.swift

```diff
@@ -5,10 +5,14 @@
 //  Copyright © 2022 Checkout. All rights reserved.
 //
 
+import Foundation
+
 extension Constants {
 
     enum Style {
 
+      static let animationLength: TimeInterval = 0.25
```

Comment:
**Interesting fact 🤓**

 
TimeInterval is actually a **Double**
```
public typealias TimeInterval = Double
```
***
But whilst Double can be inferred by compiler without any import (probably CoreFoundation), the above mentioned typealias is actually declared within Foundation -> NSDate (declared as a global alongside the NSDate object declaration, hence type is TimeInterval and not NSDate.TimeInterval).
***
And specifically here, if we declare
```swift
static let animationLength = 0.25
```
an import to Foundation would not be added, as type gets automatically inferred to Double. But at use location it would automatically be handled as the typealias if that's what's needed.

Reply:
I'd prefer to use TimeInterval here so that if this alias was ever changed then we would automatically move with it